### DISCUSSION TITLE
Using machine learning to anticipate tipping points

### DISCUSSION THREAD

**Discussion by tomfid** [Data4DM/BayesSD#14] [people; relating]
Interesting new paper ... I haven't fully absorbed, but posting for general interest.

https://arxiv.org/pdf/2207.00521.pdf

**Comment by hyunjimoon** [comment]
Related to tipping points (+ also structural piror) the following paper is highly interesting.
[Kunsch_SysDynStockPrice.pdf](https://github.com/Data4DM/BayesSD/files/9625081/Kunsch_SysDynStockPrice.pdf)

I wish to discuss following quotes from the paper @tomfid 
- equilibrium surface of the stock return folded into a cusp catastrophe.
- explicative power of this approach is in the tradition of catastrophe theory strictly limited to discontinuous changes in the evolution of prices: crashes or price upsurges.
- chaos theory in discrete non-linear systems: goal-seekers (alpha, who stabilize market) vs beta investors, is in phase with the price trend. 
- Combining the two profIles to define an iterative I-D map of the type p(t+I) = t1p(t)] one can achieve different price evolution patterns depending on the proportion of the two types of investors.
- **Bullish or bearish patterns** are generated whenever the I-D mapping has a **stable fixed point.**

Specific questions are:
- why would they have extended from two to three "extending the discrete Day's model, which had only two such families"? For the purpose of presenting structural prior, would alpha and betaST enough (or alpha and betaLT)?

I made two mdl files for each behavior (beta needs to be fixed).
![image](https://user-images.githubusercontent.com/30194633/191819360-824019a8-e446-4af7-98fa-4dd566763644.png)

**Reply by hyunjimoon** [reply]
The differential equation variable for mixture model can be connected to  Decision rule part of the figure from BD textbook ![image](https://user-images.githubusercontent.com/30194633/191897722-c1c1866c-988f-46b4-a7d6-07a2916e3d43.png)

**Reply by hyunjimoon** [reply]
- bounded rationality related to oscillation prevention?
- boom and bust related to human-behavior-based oscillation?
- attractors in stock management, beer game?
- actionable workflow but time need not be homogenous:
![image](https://user-images.githubusercontent.com/30194633/191899545-c05d8f6f-4de0-4630-a959-e6f49165a781.png)
- behavior of forecasting
![image](https://user-images.githubusercontent.com/30194633/191901723-ce6d0da9-6391-4614-ac5f-f21882b212b1.png)
- does differential equation variable encoding change behavior validation to behavior verification?

**Comment by hyunjimoon** [comment]
> Interesting new paper ... I haven't fully absorbed, but posting for general interest.
> 
> https://arxiv.org/pdf/2207.00521.pdf

From the paper, I wish to highlight the following "performance of a ML model can be highly sensitive to the combination of the choice of hyperparameters used to create and train it, and the task the model is asked to perform"

**Reply by hyunjimoon** [reply]
Core spirit seems relevant to #43, judging from "successful prediction in these examples is the new hyperparameter optimization scheme introduced in Sec. 2.2.".
